.github/workflows/unittest.yml

```diff
@@ -23,8 +23,11 @@ jobs:
         run: |
           pip install tensorflow==${{matrix.tf-version}}
           pip install -e .[test]
-      - name: Test with pytest
-        run: pytest . -n auto --cov=larq_zoo --cov-report=xml --cov-config=.coveragerc
+      - name: Test data and models with pytest
+        run: pytest tests/{data_test.py,models_test.py} -n auto --cov=larq_zoo --cov-report=xml --cov-config=.coveragerc
+      - name: Test training with pytest
+        if: matrix.python-version == '3.7' && matrix.tf-version == '2.1.0'
+        run: pytest tests/train_test.py -n auto --cov=larq_zoo --cov-report=xml --cov-config=.coveragerc --append-cov
```

Comment:
Sorry about that, looks like it was introduced [here](https://github.com/pytest-dev/pytest-cov/commit/c7b8b2ee07c019389af61f81c939af7f2c146814), but has been renamed: https://pytest-cov.readthedocs.io/en/latest/config.html
```suggestion
        run: pytest tests/train_test.py -n auto --cov=larq_zoo --cov-report=xml --cov-config=.coveragerc --cov-append
```